src/ui/proxy/interceptors.go

```diff
@@ -207,9 +207,10 @@ func (vh vulnerableHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request)
 		return
 	}
 	imageSev := overview.Sev
-	if imageSev > int(projectVulnerableSeverity) {
-		log.Debugf("the image severity is higher then project setting, failing the response.")
-		http.Error(rw, marshalError("The image scan result doesn't pass the project setting.", http.StatusPreconditionFailed), http.StatusPreconditionFailed)
+	if imageSev >= int(projectVulnerableSeverity) {
+		log.Debugf("the image severity: %s is higher then project setting: %s, failing the response.", clair.ParseHarborSev(imageSev), clair.ParseHarborSev(projectVulnerableSeverity))
+		http.Error(rw, marshalError(fmt.Sprintf("The image scan result: %s is not slower than the project setting: %s.", clair.ParseHarborSev(imageSev), clair.ParseHarborSev(projectVulnerableSeverity)),
```

Comment:
Let's rephrase it to "The severity of the vulnerability of the image: %s is not below the threshold of project setting: %s."